Calibration/EcalCalibAlgos/src/miscalibExample.cc

```diff
@@ -87,6 +59,7 @@ void miscalibExample::endJob() {
 
   scEnergy->Write();
   f.Close();
+  scEnergy->~TH1();
```

Comment:
Why not simply `delete scEnergy;`?

Reply:
No particular reason. What's the difference between deleting the pointing and calling the destructor?

Reply:
I am not sure if calling the destructor actually also deallocates the memory.
Delete on a class object calls the destructor and then deallocates the memory.